great-idea/css/services.css

```diff
@@ -0,0 +1,271 @@
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/*  HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+	margin: 0 auto;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+	
+	margin-bottom: 15px;
+	font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+}
+
+/* Your code starts here */
+
+/* ************************************************
+*                    CONTAINER                    *
+***************************************************/ 
+.container {
+	max-width: 880px;
+	margin: 0 auto;
+}
+
+/* ************************************************
+*                       NAV                       *
+***************************************************/ 
+nav {
+	display: flex;
+	justify-content: space-between;
+    align-items: center;
+
+    margin-top: 28px;
+    margin-bottom: 20px;
+}
+
+nav > a {
+	font-size: 14px;
+	color: #B3B3B3;
+	text-decoration: none;
+}
+
+/* ************************************************
+*                  SERVICES HEADER                *
+***************************************************/ 
+.service-header-container {
+    border-bottom: 1px solid black;
+}
+
+.services-header {
+    padding: 30px 20px 50px 20px;
+}
+
+h1 {
+    font-size: 28px;
+}
+
+/* ************************************************
+*               SERVICES (MAIN CONTENT)           *
+***************************************************/ 
+.services-container {
+    display: flex;
+    flex-flow: row wrap; 
+    justify-content: space-between;
+    align-content: space-evenly;
+
+    min-height: 920px;
+    border-bottom: 2px solid black;
+}
+
+.services-container section {
+    width: 48%;
+
+    border: 1px solid black;
+    padding: 20px;
+
+    background-color: #F2F1F2;
+}
+
+.services-container section p {
+    margin-top: 20px;
+    line-height: 20px;
+}
+
+.services-container section a {
+    display: inline-block; /* opted out of flex module to center the link text, seemed uneccessary */
```

Comment:
Changing it to `display: inline-block;` is a good play because you need to resize the element, but remember that flex item status only goes one deep. In your case, the `div.services-container` is where you declared `display: flex;`, so the `section`'s are flex items. However, each section's children are not flex items, so the `p` and `a` are still `display: inline;`.

Reply:
Correct: the debate I was having though was whether to _additionally_ apply `display: flex` on `.services-container section a`, then use `justify-content: center` and `align-items: center` to center the text (in addition to assigning a fixed `height` to the anchor).

I just felt like that was kind of "flex-overkill" when display-inline would have worked just fine. 

I feel like flex should be used to position larger groups of elements on the page --- then as you make your way inward into each child element (past the flex-item children) simply use regular box-model properties unless flex has some justifiable use-case --- perhaps to position another nested group of elements.

In this case, applying `display: flex` on the anchor for the simple positioning of text didn't seem warranted at all. So I opted for `display: inline`